### ISSUE TITLE
Dependencies in server and ui will not be discovered

### ISSUE THREAD

**Issue by Spatial-R** [rstudio/shinyapps#145]
When depolying the shiny, i come across the error:  
<pre>
1: In value[[3L]](cond) :
  Failed to parse C:/Users/Administrator/AppData/Local/Temp/RtmpisPIpA/file6a82b72273b/App/server.R ; dependencies in this file will not be discovered.
2: In value[[3L]](cond) :
  Failed to parse C:/Users/Administrator/AppData/Local/Temp/RtmpisPIpA/file6a82b72273b/App/ui.R ; dependencies in this file will not be discovered.
3: In FUN(c("rmarkdown", "shiny", "leaflet", "ShinyDash")[[4L]], ...) :
  Package 'leaflet' not available in repository or locally
4: In FUN(c("rmarkdown", "shiny", "leaflet", "ShinyDash")[[4L]], ...) :
  Package 'ShinyDash' not available in repository or locally
</pre>

 "leaflet" and "ShinyDash" pakcage was build using the "devtools::install_github".  Session if
<pre>
Session info -------------------------------------------------------------------------------
 setting  value                                              
 version  R version 3.1.3 (2015-03-09)                       
 system   i386, mingw32                                      
 ui       RStudio (0.99.384)                                 
 language (EN)                                               
 collate  Chinese (Simplified)_People's Republic of China.936
 tz       Asia/Taipei                                        

Packages -----------------------------------------------------------------------------------
 package     * version  date       source                                
 devtools    * 1.7.0    2015-01-17 CRAN (R 3.1.2)                        
 digest      * 0.6.8    2014-12-31 CRAN (R 3.1.2)                        
 htmltools   * 0.2.6    2014-09-08 CRAN (R 3.1.2)                        
 htmlwidgets * 0.3.3    2015-04-28 Github (ramnathv/htmlwidgets@cf6b90b) 
 httpuv      * 1.3.2    2014-10-23 CRAN (R 3.1.2)                        
 httr        * 0.6.1    2015-01-01 CRAN (R 3.1.2)                        
 leaflet       0.0.11   2015-04-25 Github (rstudio/leaflet@6bd3cb8)      
 magrittr    * 1.5      2014-11-22 CRAN (R 3.1.2)                        
 mime        * 0.3      2015-03-29 CRAN (R 3.1.3)                        
 R6          * 2.0.1    2014-10-29 CRAN (R 3.1.2)                        
 Rcpp        * 0.11.5   2015-03-06 CRAN (R 3.1.3)                        
 RJSONIO     * 1.3-0    2014-07-28 CRAN (R 3.1.2)                        
 rstudioapi  * 0.2      2014-12-31 CRAN (R 3.1.2)                        
 shiny       * 0.11.1   2015-02-11 CRAN (R 3.1.3)                        
 ShinyDash     0.0.1    2015-04-25 Github (trestletech/ShinyDash@daffe35)
 stringr     * 0.6.2    2012-12-06 CRAN (R 3.0.1)                        
 XML         * 3.98-1.1 2013-06-20 CRAN (R 3.0.1)                        
 xtable      * 1.7-4    2014-09-12 CRAN (R 3.1.2)     
</pre>

**Comment by kevinushey** [comment]
What happens if you try to manually call `parse("C:/Users/Administrator/AppData/Local/Temp/RtmpisPIpA/file6a82b72273b/App/server.R")` -- does that succeed? Can you run the application locally?

It's possible that this is an encoding problem -- if this is being called through `packrat` we might need to use similar `checkEncoding` as done in `rmarkdown`.

**Comment by Spatial-R** [comment]
Yes, i can run the shiny locally!  Maybe there are indeed some encoding problems beacuse some chinese works are in my server,ui and origin dataset.  How could i solve the problem and let the application succeed in the shinyapps?

**Comment by dxander** [comment]
I have a similar issue.  I can run the app on my local computer but when I load it to shinyapps.io it does not run completely.  I see the user interface but not the graphs.
